tests/testthat/test-b-buildmodel.R

```diff
@@ -65,7 +65,7 @@ test_that("DWLS build signature matrix works", {
 })
 
 test_that("CIBERSORTx build signature matrix works", {
-  set_cibersortx_credentials(Sys.getenv("CIBERSORTX_EMAIL"), Sys.getenv("CIBERSORTX_TOKEN"))
+  set_cibersortx_credentials("lorenzo.merotto@studenti.unipd.it", "721a387e91c495174066462484674cb8")
```

Comment:
This should get the yoken from the repo secrets

Reply:
I'll try that out!

Reply:
How would you implement it? I don't really know where to store the credentials